src/Microsoft.Extensions.Testing.Abstractions/DIA/DiaDataSource.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
@piotrpMSFT Can we upload all the DIA code as its own nuget package and reference it? This is some unnecessary bloat in our repo.